modules/installation-configuration-parameters.adoc

```diff
@@ -499,16 +499,9 @@ ifdef::azure[]
 |====
 |Parameter|Description|Values
 
-|`machines.platform.azure.type`
-|The Azure VM instance type.
-|VMs that use Windows or Linux as the operating system. See the
-link:https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-supported-os?view=azs-1908[Guest operating systems supported on Azure Stack]
-in the Azure documentation.
-
 |`machines.platform.azure.osDisk.diskSizeGB`
```

Comment:
The PR https://github.com/openshift/installer/pull/2367 was bumping up controlPlane's disk size. And I even do not know install-config.yaml has `machines` field. I think here should be `controlPlane.platform.azure.osDisk.diskSizeGB`